src/why3prover.ml

```diff
@@ -0,0 +1,79 @@
+(** Calling provers in Why3 *)
+
+open Timed
+
+(** [!current_prover] contains the name of the current prover (can be changed
+    by using the set prover <name> command). *)
+let current_prover : string ref = ref "Alt-Ergo"
+
+(** [time_limit] is the time limit (in seconds) of a prover while finding a
+    proof. *)
+let time_limit : int ref = ref 10
+
+(** [config] read the config file of Why3 that is installed in the machine.
+    the default path is [~/.why3.conf]. more information could be found in
+    http://why3.lri.fr/api/Whyconf.html *)
+let config : Why3.Whyconf.config = Why3.Whyconf.read_config None
```

Comment:
Please rename it into why3_config.

Reply:
Renamed in a29414085332857be96b62ab67a96960b46d746d.